### ISSUE TITLE
Release TwistedCore-2.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2329]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2329|
|Type|enhancement|
|Created|2006-12-25 01:24:57Z|

Yay

<details><summary>Searchable metadata</summary>

```
trac-id__2329 2329
type__enhancement enhancement
reporter__radix radix
priority__highest highest
milestone__Core_2_5 Core-2.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1167009897000000 1167009897000000
changetime__1168115084000000 1168115084000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set owner to @radix|
|-|-|

Why is this assigned to me?

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix removed owner|
|-|-|

I dunno, but now it's up for review.

branch @ releases/twisted-core-2.5.x-2329-2

please review

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @radix|
|-|-|

Sorry to reassign this when I'm not really sure what's going on.  I'll do the review if I'm wrong :).

Didn't we decide this was going to go down a bit differently today?  (i.e. "transactional" branch?)

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set owner to @glyph|
|-|-|

This branch (and the web one) was finished before that decision. If you want, I can merge them into a third single branch that I will also use for all the other releases.

Should I?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @radix|
|-|-|

Yes - I say so because I don't see any reason not to follow through on our decision.  If something new has come up though, just say so.

(Sorry for the turnaround, you know how it is, etc)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_8"></a>|@radix set status to `closed`|
|-|-|

(In [19208]) Merge releases/twisted-core-2.5.x-2329-2

Reviewer: exarkun
Author: release automation, radix
Fixes [#2329](https://github.com/twisted/twisted/issues/2329)

Release shrapnel from the 2.5.0 release of twisted-core, 
plus a minor fix to an URL in README.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
